ihmc-ros2-library/src/main/java/us/ihmc/ros2/ROS2NodeBasics.java

```diff
@@ -55,18 +65,27 @@ class ROS2NodeBasics implements ROS2NodeInterface
     * @param topicDataType       The topic data type of the message
     * @param publisherAttributes Publisher attributes created with @see{createPublisherAttributes}
     * @return a ROS 2 publisher
-    * @throws IOException if no publisher can be made
     */
    @Override
-   public <T> ROS2Publisher<T> createPublisher(TopicDataType<T> topicDataType, PublisherAttributes publisherAttributes) throws IOException
+   public <T> ROS2Publisher<T> createPublisher(TopicDataType<T> topicDataType, PublisherAttributes publisherAttributes)
```

Comment:
I removed all the throws IOException that comes from pub sub and just throw the RuntimeExceptions here.